docs/reference/FacebookVideo.md

```diff
@@ -24,6 +24,13 @@ $myVideoFileToUpload = $fb->videoToUpload('/path/to/video-file.mp4'),
 
 Partial file uploads are possible using the `$maxLength` and `$offset` parameters which provide the same functionality as the `$maxlen` and `$offset` parameters on the [`stream_get_contents()` PHP function](http://php.net/stream_get_contents).
 
+%FB(devsite:markdown-wiki:info-card {
```

Comment:
We no more use Facebook card, doc is now pure Markdown.
Please replace this by:
```markdown
> **Warning:** Uploading videos may cause a timeout. Make sure to configure your HTTP client to increase timeout time before uploading videos.
```